generator/src/test/java/com/scottlogic/deg/generator/cucumber/GreaterThan.feature

```diff
@@ -118,19 +118,19 @@ Scenario: greaterThan run against a non contradicting not greaterThan should be
        | 4   |
        | 5   |
 
-@ignore #589, #594: produces big-integer numbers rather than integer values, not sure why
+#594: order of generation to be reversed
```

Comment:
This scenario either needs:
- values to count _down_ from 4 **and** this comment and the `@ignore` to be left 
 _or_
- The comment should be removed